### DISCUSSION TITLE
There are no evaluation component in the 2.50 version

### DISCUSSION THREAD

**Discussion by yanlinchn** [langfuse/langfuse#2290] [Support] [answered]
### Describe the feature or potential improvement

I saw the demo video, there is a evaluation component in the platform. There are no evaluation component in the 2.50 version. Any idea on this?

### Additional information

![image](https://github.com/langfuse/langfuse/assets/23295101/60ebf519-d2e7-4f91-806f-f5641b863fe4)

**Comment by marcklingen** [comment, marked as answer]
Hi @yanlinchn, this is a preview feature that currently runs on some preview infra of v3 on Langfuse Cloud. Together with this upcoming version we'll release docs on how to run this feature self-hosted.
